newscoop/classes/NodeTree.php

```diff
@@ -0,0 +1,38 @@
+<?php
```

Comment:
OK

On Wed, Apr 6, 2016 at 10:06 AM, Paweł Mikołajczuk <notifications@github.com
> wrote:

> In newscoop/classes/NodeTree.php
> <https://github.com/sourcefabric/Newscoop/pull/1352#discussion_r58665901>:
>
> > @@ -0,0 +1,38 @@
> > +<?php
>
> Then please move \Node also to NewscoopBundle. I think that it's used so
> far only here.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/sourcefabric/Newscoop/pull/1352/files/3a9cc952535b0b77da2f9aea1fcbf219c43e504f#r58665901>
>